README.md

```diff
@@ -6,9 +6,9 @@
 
 In order to use this action, you need to
 
-1. [Register new GitHub App](#TBD)
-2. [Store the App's ID in your repository environment variables](#TBD)
-3. [Store the App's private key in your repository secrets](#TBD)
+1. GitHub App Credentials (ID and private key) ([Register new GitHub App](https://docs.github.com/en/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app))
+2. [Store the App's ID in your repository environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows)
+3. [Store the App's private key in your repository secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository)
```

Comment:
```suggestion
1. GitHub App Credentials (ID and private key) ([Register new GitHub App](https://docs.github.com/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app))
2. [Store the App's ID in your repository environment variables](https://docs.github.com/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows)
3. [Store the App's private key in your repository secrets](https://docs.github.com/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository)
```

Remove the language code from the URLs so users are directed to the language set in their browser.